designer/client/src/components/toolbarSettings/buttons/LinkButton.tsx

```diff
@@ -1,7 +1,7 @@
 import React, { ComponentType } from "react";
 import FallbackIcon from "../../../assets/img/toolbarButtons/link.svg";
 import { PlainStyleLink } from "../../../containers/plainStyleLink";
-import ToolbarButton from "../../toolbarComponents/ToolbarButton";
+import { ToolbarButton } from "../../toolbarComponents/toolbarButtons/index";
```

Comment:
Something is wrong with this import `toolbarButtons/index` it should be just `toolbarButtons`